stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/adapter/viewholder/ChannelListItemViewHolder.kt

```diff
@@ -94,8 +96,11 @@ public class ChannelListItemViewHolder(itemView: View) : BaseChannelListItemView
                 }
 
                 if (lastMessageChanged) {
-                    configureLastMessageLabel(channel)
-                    configureLastMessageTimestamp(channel)
+                    channel.getLastMessage().let { lastMessage ->
+                        configureLastMessageLabel(channel, lastMessage)
+                        configureLastMessageTimestamp(lastMessage)
```

Comment:
This logic will also be needed for `MessagePreviewView` which is used in Search results and Mentions, but we can figure out how to extract it later, if you don't want to do that on this PR.

Reply:
Let's extract it later :)